metrics/web/test/dashboard/presentation/state/project_metrics_notifier_test.dart

```diff
@@ -669,6 +669,34 @@ void main() {
         );
       },
     );
+
+    test(
+      ".selectProjectGroup() resets project group dropdown items and selected project group to null if project groups are null",
```

Comment:
```suggestion
      ".setProjectGroups() resets project group dropdown items and selected project group to null if project groups are null",
```
And seems like this test case lacks formatting.